package.json

```diff
@@ -9,8 +9,9 @@
     "atom": ">0.50.0"
   },
   "dependencies": {
-    "atom-linter": "^3.1.1",
-    "atom-package-deps": "^2.0.1"
+    "atom-linter": "^3.4.0",
+    "atom-package-deps": "^2.0.1",
```

Comment:
You can safely bump this up to the latest, the only change would be removing the name from the `install()` call inside activate.

(You can even leave it in, but it's no longer necessary)